vaadin-testbench-core-junit5/src/main/java/com/vaadin/testbench/AbstractBrowserTestBase.java

```diff
@@ -42,8 +42,8 @@
 import com.vaadin.testbench.commands.TestBenchCommands;
 
 /**
- * A superclass with helper methods to aid TestBench developers create a JUnit 5
- * based tests.
+ * A superclass with helper methods to aid TestBench developers create a JUnit
+ * 5+ based tests.
```

Comment:
JUnit 5+ entries could be auto-replaced by some template mechanism.
Or we can leave it like "JUnit 5/6"